R/parse.R

```diff
@@ -168,12 +170,12 @@ parse_connect_rfc3339 <- function(x) {
   })
 }
 
-vec_cast.POSIXct.double <- function(x, to, ...) {
+vec_cast.POSIXct.double <- function(x, to, ...) { # nolint: object_name_linter
   warn_experimental("vec_cast.POSIXct.double")
```

Comment:
I was confused by the PR title: I thought it meant you were going to remove all of these chatty/panicky warnings.

Reply:
If I remove the lint, this error is displayed: 
```
Variable and function name style should match snake_case, symbols or CamelCase. lintr(object_name_linter)
```

I believe this is a false-positive find as the name before the `.` is valid and the lint should be suppressed.

Reply:
With this PR's changes, no warnings are being displayed within the PR files after GitHub Actions runs. 

(Resolving)

Reply:
Ah! Makes sense. Thank you!

Yes. ... Different code quality warnings that should be surfaced / fixed.